### PR TITLE
Disable username/pwd login if google login enabled

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/pages/Login/components/LoginError.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const LoginError = ({ location, match }) => (
+  <Segment textAlign="center" padded="very" basic>
+    <Message error compact>
+      {ERROR_MESSAGES[match.params.error] || `Unknown error occured: ${match.params.error}`}
+    </Message>
+    <div>
+      <Button as="a" href={`${GOOGLE_LOGIN_URL}${location.search}`} primary>
+        <Icon name="google" />
+        Sign in to seqr
+      </Button>
```
Comment (ShifaSZ) [inline ui/pages/Login/components/LoginError.jsx:32]: A little weird to have a Google sign-in button here. How about just a link to the `landing page` or `front page`?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
A general question: is it okay to remove username/password login for local installation of seqr while enabling Google auth? If yes, the PR looks good.

File: seqr/views/apis/auth_api.py [hunk 2]
```diff
@@ -4,20 +4,24 @@
...
-from seqr.views.utils.permissions_utils import user_is_data_manager
 from seqr.views.utils.terra_api_utils import google_auth_enabled, remove_token
+from settings import LOGIN_URL
 
 logger = SeqrLogger(__name__)
 
 
 def login_view(request):
+    if google_auth_enabled():
+        raise PermissionDenied('Username/ password authentication is disabled')
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/auth_api.py:23]: no, they will have to use either one or the other

File: ui/pages/Login/components/LoginError.jsx (hunk 1 above)
Comment (hanars) [reply ui/pages/Login/components/LoginError.jsx:32]: so you only get to this page if you tried to log in and it failed. So in that case you want the ability to log in again. A common case would be you clicked the wrong google account by accident and now want to try again with the correct google account

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**